packages/core/src/styles/layout/_type-grid.scss

```diff
@@ -2,24 +2,16 @@
 // This software is released under MIT license.
 // The full license information can be found in LICENSE in the root directory of this project.
 
-@mixin generateGaps($breakpoint: null) {
-  @each $size, $sizeValue in $cds-layout-sizes {
-    [cds-layout~='grid'][cds-layout~='gap#{$breakpoint}:#{$size}'] {
-      grid-gap: #{$sizeValue};
-    }
-  }
-}
-
 @mixin generateColumnsAndRows($breakpoint: null) {
   @for $i from 1 through $cds-token-layout-grid-cols-static {
     // implicit columns/rows
-    [cds-layout~='cols#{$breakpoint}:#{$i}'] > *,
-    [cds-layout~='cols#{$breakpoint}:#{$i}']::slotted(*) {
+    [#{$layout}*='#{$cols}#{$breakpoint}:#{$i}'] > *,
+    [#{$layout}*='#{$cols}#{$breakpoint}:#{$i}']::slotted(*) {
```

Comment:
Selector names are in one file now which makes it easier to test optimizations and allow overrides. Example some systems may have strict `data-*` prefix requirements. The `*` attr selector also has a noticeable perf improvements on bundle compression size and runtime rendering.